dev/breeze/pyproject.toml

```
@@ -81,7 +81,7 @@ name = "airflow_breeze"
 
 [tool.black]
 line-length = 110
-target-version = ['py37', 'py38', 'py39', 'py310']
+target-version = ['py37', 'py38', 'py39', 'py310', 'py311', 'py312']
```

We can remove `py37` since it is not supported by Airflow
```suggestion
target-version = ['py38', 'py39', 'py310', 'py311', 'py312']
```